Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Controller.cs

```diff
@@ -49,6 +51,32 @@ partial class Controller : ItemComponent, IServerSerializable
         private Item focusTarget;
         private float targetRotation;
 
+        [InGameEditable, Serialize("1", IsPropertySaveable.Yes, description: "The signal sent when the condition is met (if empty, no signal is sent).", alwaysUseInstanceValues: true)]
+        public string Output
+        {
+            get { return output; }
+            set
+            {
+                if (value == null) { return; }
+                output = value;
+                //reactivate (we may not have been previously sending a signal, but might now)
+                IsActive = true;
+            }
+        }
+
+        [InGameEditable, Serialize("0", IsPropertySaveable.Yes, description: "The signal sent when the condition is not met (if empty, no signal is sent).", alwaysUseInstanceValues: true)]
+        public string FalseOutput
+        {
+            get { return falseOutput; }
+            set
+            {
+                if (value == null) { return; }
+                falseOutput = value;
+                //reactivate (we may not have been previously sending a signal, but might now)
+                IsActive = true;
```

Comment:
There's no need to reactivate if the value of the `falseOutput` variable doesn't change (i.e. if `value == falseOutput`).

Reply:
Added an additional check to both `output` and `falseOutput` in [af157a7](https://github.com/Regalis11/Barotrauma/pull/10419/commits/af157a79551d2add3ae4bbc5050212d39b18e77d).